requirements.txt

```diff
@@ -9,4 +9,4 @@ ipaddress>=1.0.17;python_version=="2.7"  # PSF
 websocket-client>=0.32.0,!=0.40.0,!=0.41.*,!=0.42.* # LGPLv2+
 requests # Apache-2.0
 requests-oauthlib # ISC
-adal>=1.0.2 # MIT
+adal>=1.0.2;python_version=='2.7' # MIT
```

Comment:
This will cause adal to be installed automatically for Python 2.7.*, but not Python 3.* with setuptools. Since ```python-base``` does not have its own requirements.txt, it may cause ```kube_config```to fail.